web/css/list.css

```diff
@@ -0,0 +1,23 @@
+.list-group-item {
```

Comment:
Nice work with the CSS changes. I like how list and modal are broken out into separate files; keeps them decoupled like the JS. I'm wondering if we should have a sort of components sub-folder at some point for these type of elements.

Reply:
I like it.